src/common/libjob/job_info.h

```diff
@@ -0,0 +1,36 @@
+/************************************************************\
+ * Copyright 2018 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef _FLUX_CORE_JOB_EVENTLOG_H
```

Comment:
I forgot where we left the discussion, but I think @grondo earlier proposed just adding this to job.h rather than creating job_info.h, job_types.h, etc., just to keep the number of installed headers under control.

Reply:
That was discussion on `job_util.h`.  I felt `job_info.h` warranted a new file given future additions?  But perhaps we are thinking too far ahead given your comment below on what the API should look like.  I'll just squash it all into `job.h` for the time being.